### PR TITLE
Webui command

### PR DESCRIPTION
**Pull request by daniviga**
Add a command to start the WebUI using `oq`. The django development server is started in foreground (that's a feature, not a bug) on localhost:8800. Meant to be used with `multi_user = false` but works also if true.

I want this to simplify the start of the WebUI in the 'opt' packages and also in development, it's shorter than `python -m openquake.server.manage ecc...` and it also checks that DbServer is running. If the DbServer is not started and `manage.py runserver` is fired no errors appear on the screen and the django server remains locked until `CTRL-C` is issued.

This feature is currently not tested by CI.

Closes https://github.com/gem/oq-installers/issues/20

### CODE REVIEW

File: openquake/commands/webui.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import subprocess
+from openquake.risklib import valid
+from openquake.commonlib import sap
+from openquake.engine import config
+from openquake.server.dbserver import get_status
+from openquake.commands.dbserver import runserver
+
+
+def rundjango(subcmd, host=''):
+    args = [sys.executable, '-m', 'openquake.server.manage', subcmd]
```
Comment (micheles) [inline openquake/commands/webui.py]: I would change this to

hostport='127.0.0.1:8800'

**Comment by daniviga** [comment]
Ping @micheles

File: openquake/commands/webui.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from openquake.server.dbserver import get_status
+from openquake.commands.dbserver import runserver
+
+
+def rundjango(subcmd='runserver', hostport='127.0.0.1:8800'):
+    args = [sys.executable, '-m', 'openquake.server.manage', subcmd]
+    if host:
+        args.append(hostport)
+    subprocess.call(args)
+
```
Comment (micheles) [inline openquake/commands/webui.py]: I would replace `subcmd='runserver' -> subcmd`; also `if host -> if hostname`

File: openquake/commands/webui.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    if dbstatus == 'not-running':
+        if valid.boolean(config.get('dbserver', 'multi_user')):
+            sys.exit('Please start the DbServer: '
+                     'see the documentation for details')
+        runserver()
+
+    if cmd == 'start':
+        rundjango()
+    elif cmd == 'syncdb':
+        rundjango(cmd)
```
Comment (micheles) [inline openquake/commands/webui.py]: I would write `rundjango('runserver')` and then `rundjango('syncdb')`

File: openquake/commands/webui.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+parser = sap.Parser(webui)
+parser.arg('cmd', 'webui command',
+           choices='start syncdb'.split())
```
Comment (micheles) [inline openquake/commands/webui.py]: Please add a line
`parser.arg('hostport', 'a string of the form <hostname:port>')`

**Comment by micheles** [comment]
LGTM

**Pull request merged**